src/Tempest/Database/src/Migrations/CreateMigrationsTable.php

```diff
@@ -18,13 +19,13 @@ public function getName(): string
 
     public function up(): QueryStatement|null
     {
-        return (new CreateTableStatement('Migration'))
+        return (new CreateTableStatement(Model::table()->tableName))
```

Comment:
Couldn't you use `CreateTableStatement::forModel(Model::class)`?

Is there a reason you import `Migration` as `Model`?

Reply:
> Yes, it's because the Migration interface is already imported. I chose Model since the Migration class is a model, but we can rename it to something else

Aaah yes, I forgot about that. I really hate the name collision, but don't know what to do about it. If you have suggestions, I welcome them! Not blocking this PR though.